routes/utils/request-to-bearer-token.js

```diff
@@ -1,8 +0,0 @@
-module.exports = toBearerToken
-
-function toBearerToken (request) {
-  if (!request.headers.authorization) {
-    return ''
-  }
-  return request.headers.authorization.substr(7)
```

Comment:
I wouldn’t make it configurable, but we could make it more clear with something like `'session '.length`